src/module/StaticPage/config/module.config.php

```diff
@@ -0,0 +1,36 @@
+<?php
+/**
+ * @author    Benjamin Knorr (benjamin@serlo.org]
+ * @copyright 2017 by www.serlo.org
+ * @license   LGPL
+ * @license   http://www.gnu.org/copyleft/lesser.html GNU Lesser General Public License (LGPL]
+ */
+namespace StaticPage;
+
+return [
+    'di'              => [
+        'allowed_controllers' => [
+            __NAMESPACE__ . '\Controller\StaticPageController'
+        ],
+        'definition'          => [
+            'class' => [
+                __NAMESPACE__ . '\Controller\StaticPageController' => []
+            ]
+        ]
+    ],
+    'router'          => [
+        'routes' => [
+            'spenden' => [
+                'type'          => 'literal',
+                'options'       => [
+                    'route'    => '/spenden',
```

Comment:
this will be "spenden" on both en and us

Reply:
yes... is there a way to only add it for `de.serlo.org` using the router?

Reply:
I believe so, best check in the ZF2 router docs